osbs/build/build_request.py

```diff
@@ -660,6 +661,25 @@ def render_bump_release(self):
 
         self.dj.dock_json_set_arg(phase, plugin, 'hub', hub)
 
+    def render_fetch_worker_metadata(self):
+        if not self.dj.dock_json_has_plugin_conf('exit_plugins',
+                                                 'fetch_worker_metadata'):
+            return
+
+        if self.spec.kojihub.value:
```

Comment:
`fetch_worker_metadata` takes no parameters. Don't think we need render method at all.